src/PowerSimulations.jl

```diff
@@ -7,12 +7,13 @@ module PowerSimulations
 # Base Models
 export Simulation
 export OperationsProblem
+export OperationsProblemResults
 export OperationsProblemTemplate
 export InitialCondition
 export SimulationProblems
 export SimulationSequence
 export SimulationResults
-export StageResults
+export ProblemResults
```

Comment:
Do we want to export ProblemResults or make it an internal object?

Reply:
It is a type that users will see and use. It's possible that we will make a `ProblemResults` constructor in the future that is similar to `SimulationResults`. I would vote for keeping it exported but am fine either way.